app/assets/javascripts/application.js

```diff
@@ -16,7 +16,20 @@
 //= require bootstrap
 //= require bootstrap-sprockets
 //= require turbolinks
-//= require growlyflash
 //= require moment
```

Comment:
Moment and bootstrap-datetimepicker have been giving errors such as 
`couldn't find file 'moment' with type 'application/javascript'` on my dev